dev/bots/test.dart

```diff
@@ -176,6 +178,55 @@ Future<bq.BigqueryApi> _getBigqueryApi() async {
   }
 }
 
+List<List<String>> _partitionToolTests() {
+  final List<File> pending = <File>[];
+  final String toolTestDir = path.join(toolRoot, 'test');
+  for (FileSystemEntity entity in Directory(toolTestDir).listSync(recursive: true)) {
+    if (entity is File && entity.path.endsWith('_test.dart')) {
+      pending.add(entity);
+    }
+  }
+  final List<String> groupA = <String>[];
```

Comment:
I think this code can probably be simplified by using List.map, List.take, and List.skip.